packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/progress-bar';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlDashboardHeaderGlobalStyles from '../styles/global/cxl-dashboard-header-css.js';
+import cxlDashboardHeaderStyles from '../styles/cxl-dashboard-header-css.js';
+
+@customElement('cxl-dashboard-header')
+export class CXLDashboardHeaderElement extends LitElement {
+  static get styles() {
+    return [cxlDashboardHeaderStyles];
+  }
+
+  @property({ type: String }) title = 'Hello';
+
+  @property({ type: String }) subtitle = 'My dashboard';
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) notificationCount = '6';
+
+  @property({ type: String }) updateTitle = "What's new in CXL";
```

Comment:
Keep it consistent with `notificationCount`

```suggestion
  @property({ type: String }) notificationsTitle = "What's new in CXL";
```